### PR TITLE
Adds "Using a PWA in your Android app" article

### PR DESCRIPTION
**Pull request by andreban**
- Adds the article
- Adds `andreban` to contributors, with photos
- Updates `/progressive-web-appa` to list the new article

### CODE REVIEW

**Comment by andreban** [comment]
@jpmedley FYI

File: src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+nightly version. Other browsers are expected to follow soon.
+
+### Quality Criteria
+
+Web developers should use a Trusted Web Activity when they want to include web content in an
+Android app.  Web content in a Trusted Web Activity must meet PWA install quality criteria and
+additional Android-specific criteria such as [policy compliance][9].
+
+<figure class="w-figure w-figure--center">
+  <img src="./lh-score.png" style="max-width: 100%;"
```
Comment (jpmedley) [inline src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md]:
```suggestion
  <img src="lh-score.png" style="max-width: 100%;"
```

Comment (jpmedley) [reply src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md]: The link was broken in the [current preview](https://deploy-preview-2388--web-dev-staging.netlify.com/using-a-pwa-in-your-android-app/). This is how its done in other articles.

File: src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+title: Using a PWA in your Android app
+authors:
+  - andreban
+date: 2020-03-19
+updated: 2020-03-19
```
Comment (jpmedley) [inline src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md]:
```suggestion
date: 2020-03-19
```

**Comment by andreban** [comment]
Thanks for review and the sanity check. Added the suggestions and fixed the remaining ones in a commit.

**Comment by kaycebasques** [comment]
@andreban please resolve the git conflict in `src/site/_data/contributors.js`

File: src/site/_data/contributors.js [hunk 3]
```diff
@@ -1723,6 +1723,22 @@ const contributors = {
...
+      name: "Google",
+      unit: "Developer Relations",
+    },
+    country: "UK",
+    homepage: "https://bandarra.me/",
+    twitter: "andreban",
+    github: "andreban",
+    description: {
+      en: "Andre is a Developer Advocate",
+    },
```
Comment (kaycebasques) [inline src/site/_data/contributors.js]:
```suggestion
    },
  },
```

File: src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+```
+
+After running this command, a file called `app-release-signed.apk` will be available in the root
+directory for the project. This is the file that will be [uploaded to the Play Store][11].
+
+### PWABuilder
+
+[PWABuilder][12] helps developers transform existing websites into Progressive Web Apps. It also
+integrates with Bubblewrap to provide a GUI interface to wrap those PWAs into an Android app.
+The PWABuilder has put together a [great blog post][13] on how to generate an Android application
```
Comment (kaycebasques) [inline src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md]:
```suggestion
The PWABuilder team has put together a [great blog post][13] on how to generate an Android application
```

File: src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+
+If you are getting started with Progressive Web Apps, read
+[our guidance on how to build a great PWA][1]. For developers who already have a PWA, use
+[Lighthouse][16] to verify if it meets the quality criteria.
+
+Then, use [Bubblewrap][10] or [PWABuilder][12] to generate the Android application,
+[upload the application to a closed test channel on the Play Store][11] and pair it with the PWA
+using [Peter's Asset Link Tool][15].
+
+Finally, move the application from the closed test channel to production
```
Comment (kaycebasques) [inline src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md]:
```suggestion
Finally, move your application from the closed test channel to production!
```

File: src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+[7]: /native-file-system/
+[8]: https://developers.google.com/web/updates/2019/02/using-twa
+[9]: https://play.google.com/about/developer-content-policy/
+[10]: https://github.com/GoogleChromeLabs/bubblewrap
+[11]: https://support.google.com/googleplay/android-developer/answer/3131213?hl=en-GB
+[12]: https://pwabuilder.com/
+[13]: https://www.davrous.com/2020/02/07/publishing-your-pwa-in-the-play-store-in-a-couple-of-minutes-using-pwa-builder/
+[14]: https://developers.google.com/digital-asset-links/v1/getting-started
+[15]: https://play.google.com/store/apps/details?id=dev.conn.assetlinkstool
+[16]: /lighthouse-performance/
```
Comment (kaycebasques) [inline src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md]:
```suggestion
[16]: https://developers.google.com/web/tools/lighthouse
```

File: src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+APIs that the underlying browser does. There are also open source utilities to make implementing
+an Android app using a Trusted Web Activity even easier.  
+
+Another advantage not available in other solutions is that the container shares storage with the
+browser. Login states and users preferences are shared seamlessly across experiences.
+
+#### Browser Compatibility
+
+The feature has been available in Chrome since version 75, with Firefox implementing it in their
+nightly version. Other browsers are expected to follow soon.
```
[the remark this answers is not in the document]
Comment (andreban) [reply src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md]: 👍this makes sense.

**Comment by andreban** [comment]
I was just looking through them. All makes sense, and looks better. Thanks for the review.

**Comment by kaycebasques** [comment]
Fixes #2436

**Pull request merged**